app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
 
+  has_many :created_tests, :class_name => "Test", inverse_of: 'author'
+  has_many :tests_users
+  has_many :tests, through: :tests_users, dependent: :destroy
+
   def tests_by_level(level)
-    Test
-      .joins("JOIN user_tests ON tests.id = user_tests.test_id")
-      .where("user_id = :id AND level = :level", id: id, level: level)
+      created_tests
```

Comment:
```suggestion
  tests
```
в том задании, где добавлен этот метод речь шла о "пройденных тестах", не о "созданных тестах"
обрати внимание на код, который был раньше:
```ruby
.joins("JOIN user_tests ON tests.id = user_tests.test_id")
```
через соединительную таблицу у тебя `tests`